src/resolvers/viewsInfo.ts

```diff
@@ -46,13 +46,25 @@ class BatchedChannelViewsArgs {
   channelIdList: string[]
 }
 
+@ArgsType()
+class MostViewedCategoriesArgs {
+  @Field(() => Int, { nullable: true })
+  period?: number
```

Comment:
Let's make `period` required and validate that it's `<= 30`. Orion is publicly accessible so let's try to restrict unnecessary computing people may want to burden it with. This also applies to other queries where `period` is used.